core/src/main/java/com/linecorp/armeria/server/RoutingContext.java

```diff
@@ -48,6 +48,19 @@ public interface RoutingContext {
      */
     HttpMethod method();
 
+    /**
+     * Returns a wrapped {@link RoutingContext} which holds the specified {@link HttpMethod}.
+     */
+    default RoutingContext overrideMethod(HttpMethod method) {
```

Comment:
`withMethod`? `with*` seems like a popular choice when creating a new instance.

Reply:
I agree with withMethod is appropriate naming :)

However there is a question. I referred to below code in `RoutingContext`.
Is there reason about different naming between them?

```
default RoutingContext overridePath(String path) {
        requireNonNull(path, "path");
        return new RoutingContextWrapper(this) {
            @Override
            public String path() {
                return path;
            }
        };
    }
```

Reply:
Ok 😄 I'll apply it!